console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx

```diff
@@ -133,7 +137,7 @@ const ConnectionsToggleGroup: React.FunctionComponent<IConnectionsToggleGroupPro
                 setInput={setHostContainerInput}
               />
               <Button
-                id="cl-filter-search"
+                id="connections-togglegrp-search-addr-button"
```

Comment:
I don't think this button here searches for address. Can you clarify what exactly this button searches?